arm-sql/2015-05-01-preview/swagger/blobAuditingPolicies.json

```diff
@@ -106,120 +132,40 @@
+            "$ref": "../examples/DatabaseBlobAuditingCreateMax.json"
           }
         }
       }
     }
   },
   "definitions": {
-"Resource":{
-      "description":"ARM resource.",
-      "properties":{  
-        "id":{  
-          "readOnly":true,
-          "type":"string",
-          "description":"Resource ID."
-        },
-        "name":{  
-          "readOnly":true,
-          "type":"string",
-          "description":"Resource name."
-        },
-        "type":{  
-          "readOnly":true,
-          "type":"string",
-          "description":"Resource type."
-        }
-      },
-      "x-ms-azure-resource": true
-    },
-    "TrackedResource":{  
-      "description":"ARM tracked top level resource.",
-      "properties":{  
-        "tags":{  
-          "type":"object",
-          "additionalProperties":{  
-            "type":"string"
-          },
-          "x-ms-mutability":[  
-            "read",
-            "create",
-            "update"
-          ],
-          "description":"Resource tags."
-        },
-        "location":{  
-          "type":"string",
-          "x-ms-mutability":[  
-            "read",
-            "create"
-          ],
-          "description":"Resource location."
-        }
-      },
-      "required":[  
-        "location"
-      ],
-      "allOf":[  
-        {  
-          "$ref":"#/definitions/Resource"
-        }
-      ]
-    },
-    "ProxyResource":{  
-      "description":"ARM proxy resource.",
-      "allOf":[  
-        {  
-          "$ref":"#/definitions/Resource"
-        }
-      ]
-    },
-    "DatabaseBlobAuditingPolicy": {
-      "description": "Contains information about a database Blob Auditing policy.",
-      "type": "object",
-      "properties": {
-        "location":{  
-          "type":"string",
-          "x-ms-mutability":[  
-            "read",
-            "create"
-          ],
-          "description":"The geo-location where the resource lives"
-        },
-        "kind": {
-          "readOnly": true,
-          "type": "string",
-          "description": "Resource kind."
-        },
-        "properties": {
-          "$ref": "#/definitions/DatabaseBlobAuditingPolicyProperties",
-          "x-ms-client-flatten": true
-        }
-      },
-      "allOf": [
-        { "$ref": "#/definitions/ProxyResource" }
-      ]
-    },
     "DatabaseBlobAuditingPolicyProperties": {
-      "description": "Properties for a database Blob Auditing policy.",
+      "description": "Properties of a database blob auditing policy.",
       "type": "object",
       "properties": {
         "state": {
           "description": "Specifies the state of the policy. If state is Enabled, storageEndpoint and storageAccountAccessKey are required.",
-          "type": "string",
           "enum": [
             "Enabled",
             "Disabled"
           ],
-          "x-ms-enum":{
+          "type": "string",
+          "x-ms-enum": {
             "name": "BlobAuditingPolicyState"
```

Comment:
Not a big deal, but are you on the latest?  Generator should be adding "modelAsString": "false" here